ihmc-footstep-planning/src/main/generated-java/us/ihmc/footstepPlanning/AStarBodyPathPlannerParametersBasics.java

```diff
@@ -370,4 +371,89 @@ default void setSmootherGradientThresholdToTerminate(double smootherGradientThre
    {
       set(AStarBodyPathPlannerParameters.smootherGradientThresholdToTerminate, smootherGradientThresholdToTerminate);
    }
+
+   default void set(AStarBodyPathPlannerParametersPacket packet)
+   {
+         setCheckForCollisions(packet.getCheckForCollisions());
+         setComputeSurfaceNormalCost(packet.getComputeSurfaceNormalCost());
+         setComputeTraversibility(packet.getComputeTraversibility());
+         setPerformSmoothing(packet.getPerformSmoothing());
+
+      if (packet.getRollCostWeight() != -1.0)
+         setRollCostWeight(packet.getRollCostWeight());
```

Comment:
Why not log if it's -1? Does that mean "unset"?

Reply:
Ahhh honestly the convention is all over the place but generally we assume a -1 value means the parameter is disabled. Something like `if (parameter.getValue() > 0)` and then we do something